### ISSUE TITLE
ExcludeFromStyleCop not respected by the VS extension

### ISSUE THREAD

**Issue by MT-PL-SWF-Dev** [Visual-Stylecop/Visual-StyleCop#53]
ExcludeFromStyleCop works fine for project's build.

But when using "Run StyleCop (Rescan All)" from the VS extension on a solution level it still raises warnings, e.g.:
SA1633 : CSharp.Documentation : The file has no header, the header Xml is invalid, or the header is not located at the top of the file.

**Comment by ptittof57** [comment]
It is the AssemblyInfo.cs file ?

**Comment by MT-PL-SWF-Dev** [comment]
Yes, it is. We actually have AssemblyInfo split into two - common between projects (which is added as link) and project-specific which is added in a standard way. The issue occurs for both files.

**Comment by ptittof57** [comment]
@MT-PL-SWF-Dev I can't reproduce this issue, could you send me a zip of your solution, and projects (with AssemblyInfo only). You can remove all other cs files from your personal code this is just to try on my side

**Comment by MT-PL-SWF-Dev** [comment]
I enclose a zip with sample project and some print screens in main directory. I hope this helps.
[Test.zip](https://github.com/Visual-Stylecop/Visual-StyleCop/files/140896/Test.zip)

**Comment by ptittof57** [comment]
I found this explicit code for links:

``` csharp
                var isLinkProperty = item.Properties.Item("IsLink");

                if (isLinkProperty != null && bool.Parse(isLinkProperty.Value.ToString()))
                {
                    // The ProjectItem is a linked file to we'll assume these haven't got
                    // ExcludeFromStyleCop in the proj file
                    return true;
                }
```

I don't want to remove this code so I think the best solution is to add  a new parameter.

**Comment by MT-PL-SWF-Dev** [comment]
How does the linked file differ from a regular one? Both are tested against StyleCop violations, so I would actually assume the other way around - that it may have ExcludeFromStyleCop attribute.
